endpoints/openrtb2/auction.go

```diff
@@ -1371,16 +1372,21 @@ func (deps *endpointDeps) processStoredRequests(ctx context.Context, requestJson
 					err = fmt.Errorf("Invalid JSON in Default Request Settings: %s", Err)
 				}
 			}
-			return nil, []error{err}
+			return nil, nil, []error{err}
 		}
 		resolvedRequest = aliasedRequest
 	}
 
 	// Apply any Stored Imps, if they exist. Since the JSON Merge Patch overrides arrays,
 	// and Prebid Server defers to the HTTP Request to resolve conflicts, it's safe to
 	// assume that the request.imp data did not change when applying the Stored BidRequest.
+	impToStoredReq := make(map[string][]byte)
 	for i := 0; i < len(impIds); i++ {
 		resolvedImp, err := jsonpatch.MergePatch(storedImps[impIds[i]], imps[idIndices[i]])
+
+		impId, err := jsonparser.GetString(resolvedImp, "id")
+		impToStoredReq[impId] = storedImps[impIds[i]]
```

Comment:
Please check the error form the call to `GetString`.

Reply:
Added.